src/plugins/intel_gpu/include/intel_gpu/primitives/moe_gather.hpp

```diff
@@ -0,0 +1,62 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+#include "primitive.hpp"
+#include "intel_gpu/op/moe_compressed.hpp"
+
+namespace cldnn {
+
+/// @brief
+/// @details
+struct moe_gather : public primitive_base<moe_gather> {
+    CLDNN_DECLARE_PRIMITIVE(moe_gather)
+
+    moe_gather() : primitive_base("", {}) {}
+
+    /// @brief Constructs moe_gather primitive.
+    ///
+    /// @param id                            This primitive id.
+    /// @param input                         Input data primitive id.
+    /// @param experts_info_offsets          offset of each expert's info from the tokens_per_expert
```

Comment:
It seems that this parameter(`experts_info_offset`) is no longer used and should be removed.